File: EXAMEN LENGUAJES/programas_pascal_dosbox/matrices.pas
program SumaMatrices;
uses crt;
var
   filas, columnas, i, j : integer;
   matriz1, matriz2, suma : array[1..10, 1..10] of integer;
begin
   clrscr;
   write('Ingresa el numero de filas: ');
   readln(filas);
   write('Ingresa el numero de columnas: ');
   readln(columnas);
   writeln;
   writeln('Ingresa los valores de la primera matriz:');
   for i := 1 to filas do
      for j := 1 to columnas do
      begin
         write('Matriz1[', i, '][', j, '] = ');
         readln(matriz1[i][j]);
      end;
   writeln;
   writeln('Ingresa los valores de la segunda matriz:');
   for i := 1 to filas do
      for j := 1 to columnas do
      begin
         write('Matriz2[', i, '][', j, '] = ');
         readln(matriz2[i][j]);
      end;
   for i := 1 to filas do
      for j := 1 to columnas do
         suma[i][j] := matriz1[i][j] + matriz2[i][j];
   writeln;
   writeln('La matriz resultante (suma) es:');
   for i := 1 to filas do
   begin
      for j := 1 to columnas do
         write(suma[i][j]:4);
      writeln;
   end;
   readln;
end.

File: EXAMEN LENGUAJES/Examenes_Lenguajes/src/examenes_lenguajes/Matrices.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package examenes_lenguajes;

/**
 *
 * @author emmch
 */

 import java.util.Scanner;
public class Matrices {
   


    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);


        System.out.print("Ingresa el numero de filas: ");
        int filas = sc.nextInt();
        System.out.print("Ingresa el numero de columnas: ");
        int columnas = sc.nextInt();

        int[][] matriz1 = new int[filas][columnas];
        int[][] matriz2 = new int[filas][columnas];
        int[][] suma = new int[filas][columnas];

       
        System.out.println("\nIngresa los valores de la primera matriz:");
        for (int i = 0; i < filas; i++) {
            for (int j = 0; j < columnas; j++) {
                System.out.print("Matriz1[" + i + "][" + j + "] = ");
                matriz1[i][j] = sc.nextInt();
            }
        }

       
        System.out.println("\nIngresa los valores de la segunda matriz:");
        for (int i = 0; i < filas; i++) {
            for (int j = 0; j < columnas; j++) {
                System.out.print("Matriz2[" + i + "][" + j + "] = ");
                matriz2[i][j] = sc.nextInt();
            }
        }

       
        for (int i = 0; i < filas; i++) {
            for (int j = 0; j < columnas; j++) {
                suma[i][j] = matriz1[i][j] + matriz2[i][j];
            }
        }

       
        System.out.println("\nLa matriz resultante (suma) es:");
        for (int i = 0; i < filas; i++) {
            for (int j = 0; j < columnas; j++) {
                System.out.print(suma[i][j] + "\t");
            }
            System.out.println();
        }
    }
}

    

